docker-cloud/builds/link-source.md

```diff
@@ -3,15 +3,9 @@ aliases: /docker-cloud/tutorials/link-source/
 description: Link to your source code repository
 keywords:
 - sourcecode, github, bitbucket, Cloud
-menu:
-  main:
-    parent: builds
-    weight: -60
-title: Link to a source code repository
+title: Link Docker Cloud to a source code provider
```

Comment:
This is the long form, right? The shorter version appears nested in context in the left nav menu?